### DISCUSSION TITLE
Do statically loaded module resources need to be manually included with a custom `baseof.html` file?

### DISCUSSION THREAD

**Discussion by anoduck** [gethinode/hinode#1321] [Help] [answered]
On a production site, if the desired module's resources are loaded from `/static/`, does a custom `/layouts/_default/baseof.html` file need to be created with the same additions as in the module's `examplesite/layouts/_default/baseof.html` to load those resources? Or does hugo take care of this?

Writing the question down makes the answer fairly obvious, but still would be nice to acquire some confirmation, even if the question is silly.

**Comment by d-oit** [comment, marked as answer]
All folders, files are defined in the config.toml

https://github.com/d-oit/hinode-mod-slideshow-gallery/blob/main/config.toml


In the exampleSite you only need the reference to the module. 
https://github.com/d-oit/hinode-mod-image-lightbox-gallery/blob/main/exampleSite%2Fhugo.toml

You could see the result after the build in the vendor folder.

**Reply by anoduck** [reply]
@d-oit [config.toml](https://github.com/anoduck/mod-timelinejs/blob/main/config.toml) is pretty similar, but [hugo.toml](https://github.com/anoduck/mod-timelinejs/blob/main/exampleSite/hugo.toml) follows a different convention due to loading resources from the `/static/` folder. 

~~The static load strategy was chosen as a countermeasure to avoid Hinode's CSS styling rules from squashing module feature visibility. Apparently, they don't mix too well.~~

**Reply by anoduck** [reply]
Loading resources from static was chosen because it was the same strategy chosen in the [guide](https://gethinode.com/guides/modules/), which it was important to follow as close as possible to ensure correct methodology, and this would allow the JavaScript and CSS files to bypass being processed by Hinode and Hugo.
